### PR TITLE
New brain damage line: the scrungulartiy

### PR DESCRIPTION
**Pull request by nervere**
:cl: Nervere
add: Added the scrungulartiy brain damage line.
/:cl:

context: scrungulartiy

We must bring the divine scrungulartiy to /tg/.
Credit to Goldsteel for the original PR on /vg/.

### CODE REVIEW

File: strings/traumas.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
         "DON'T EVER TUCH ME",
         "@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS",
         "its gonna be like 9/11 all over again but insitaead is gonna be a bitc"
```
Comment (optimumtact) [inline strings/traumas.json]: you need a ,

**Comment by nervere** [comment]
Critical spelling errors fixed.

File: strings/traumas.json [hunk 2]
```diff
@@ -64,7 +64,8 @@
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
         "DON'T EVER TUCH ME",
         "@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS",
-        "its gonna be like 9/11 all over again but insitaead is gonna be a bitc"
+        "its gonna be like 9/11 all over again but insitaead is gonna be a bitc",
+		"@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungularity????"
```
Comment (subject217) [inline strings/traumas.json]:
```suggestion
		"@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????"
```
dumb headmin

**Review by Cyberboss — CHANGES_REQUESTED** [review]
^

File: strings/traumas.json [hunk 3]
```diff
@@ -64,7 +64,8 @@
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
         "DON'T EVER TUCH ME",
         "@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS",
-        "its gonna be like 9/11 all over again but insitaead is gonna be a bitc"
+        "its gonna be like 9/11 all over again but insitaead is gonna be a bitc",
+		"@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????"
```
Comment (Cyberboss) [inline strings/traumas.json]:
```suggestion
        "@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????"
```

Spaces not tabs here

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by PKPenguin321** [comment]
SCo

**Pull request merged**